qubesdbus/device_manager.py

```diff
@@ -234,7 +235,7 @@ def Detached(self, vm_obj_path):
 
 
 def device_path(vm, dev_class, ident):
-    _id = ident.replace('.', '_')
+    _id = re.sub(r"[^A-Za-z0-9_/]", "_", ident)
```

Comment:
PS: It's part of the `data` dictionary passed at the instantiation of `Device(BUS_NAME, OBJ_PATH, DATA)`